prdoc/pr_7553.prdoc

```diff
@@ -0,0 +1,15 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: Add more checks to `chill_other` tests
```

Comment:
nit: An alternative would be t0-silent for tests, however I like the PR doc description 🙏

Reply:
By `t0-silent`, you are referring to a PR tag on GitHub, correct?